internal/config/config.go

```diff
@@ -79,12 +79,17 @@ func New() *Config {
 	return c
 }
 
-// NewNoWrites initializes a new config that does not allow writes. For use in tests.
+// NewReadOnly initializes a new config that does not allow writes. For use in tests.
 // This does not migrate legacy option names to their correct config section.
-func NewNoWrites() *Config {
+func NewReadOnly() *Config {
 	return newWithOptions(true)
```

Comment:
This probably explains why the tests are all still passing: it's not really a readonly config as we mean it in our gitconfig here:

https://github.com/gopasspw/gopass/blob/d168602f770b00eb2d531ae6337c3c316ee8299c/pkg/gitconfig/config.go#L21-L22

It's actually a `Config` with `noWrites` set (that's what `newWithOptions(noWrites bool)` does) and so it's not really `ReadOnly`.

Reply:
Shall we revert this rename?

Reply:
Yeah, probably